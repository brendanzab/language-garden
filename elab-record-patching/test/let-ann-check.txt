let Id := fun (A : Type) -> A -> A;

let id : Id := fun A a => a;

id {} {}
