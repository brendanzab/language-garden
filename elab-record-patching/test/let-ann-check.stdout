<input> : {} :=
  let Id : Type := fun (A : Type) (_ : A) -> A;
  let id : Id := fun A a => a;
  id {} {}
